lib/Thumbor/Url.php

```diff
@@ -44,6 +44,11 @@ public function stringify($server, $secret, $original, $commands)
             $imgPath = $original;
         }
 
+        if(is_array($server)) {
+            $index = strlen($imgPath) % count($server);
```

Comment:
I can implement this pending the answers to @pda question about whether this should be implemented in the bundle.